### PR TITLE
testing https://github.com/jenkinsci/jenkins-test-harness/pull/782

### PR DESCRIPTION
**Pull request by jtnord**
### Submitter checklist
- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: pct.sh [hunk 1]
```diff
@@ -26,8 +26,10 @@ exec java \
 	--war "$(pwd)/target/megawar-$LINE.war" \
 	--include-plugins "${PLUGINS}" \
 	--working-dir "$(pwd)/target/pct-work" \
+	--maven-args "-Pconsume-incrementals"
```
Comment (basil) [inline pct.sh:29]: Doesn't this need a backslash at the end of the line to escape the new line?

File: pct.sh [hunk 2]
```diff
@@ -26,8 +26,10 @@ exec java \
...
 	$PCT_D_ARGS \
 	${PCT_OPTS-} \
+	-Djenkins-test-harness.version=2216.v50ea_d7db_b_e2a_
```
Comment (basil) [inline pct.sh:32]: Doesn't this need a backslash at the end of the line to escape the new line?

File: pct.sh (hunk 1 above)
Comment (timja) [reply pct.sh:29]:
```suggestion
	--maven-args "-Pconsume-incrementals" \
```

File: pct.sh (hunk 2 above)
Comment (timja) [inline pct.sh:32]:
```suggestion
	-Djenkins-test-harness.version=2216.v50ea_d7db_b_e2a_ \
```

File: pct.sh [hunk 3]
```diff
@@ -26,8 +26,10 @@ exec java \
 	--war "$(pwd)/target/megawar-$LINE.war" \
 	--include-plugins "${PLUGINS}" \
 	--working-dir "$(pwd)/target/pct-work" \
+	--maven-args "-Pconsume-incrementals" \
 	$PCT_D_ARGS \
 	${PCT_OPTS-} \
+	-Djenkins-test-harness.version=2217.v9fc3c7d9c602 \
```
Comment (jtnord) [inline pct.sh:32]: https://github.com/jenkinsci/jenkins-test-harness/pull/782

**Comment by jtnord** [comment]
PR served its purpose

**Pull request closed without merging**